src/style.css

```diff
@@ -1 +1,85 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto, sans-serif;
+  src: url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+  cursor: pointer;
+}
+
+.header {
+  /* display: flex;
+  position: sticky; */
+  top: 0;
+  text-transform: uppercase;
+  font-weight: 500;
+  background: #fff;
+}
+
+.logo {
+  background-image: url(images/logo.png);
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-position-x: 50px;
+  height: 40px;
+}
+
+.nav {
+  text-align: center;
+  /* background-color: pink; */
+}
+
+.nav__list {
+  margin: 0;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: right;
+
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin-right: 20px;
+  height: 60px;
+
+  color: #000;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.is-active::after {
+  content: "";
+  height: 4px;
+  width: 100%;
+
+  border-style: solid;
+  border-radius: 8px;
+  border-color: #00acdc;
+  background-color: #00acdc;
+}
+
+.lastLink {
+  padding-right: 0;
+}
```

Comment:
you can use the last-child pseudo class for it. no need to create a separate class just for this